ui/src/app/views/app/component.html

```diff
@@ -1,7 +1,7 @@
 <ng-template *ngIf="(user | async).isSignedIn; then signedIn else notSignedIn"></ng-template>
 <ng-template #logoImages>
   <img class="main-logo" src="assets/images/all-of-us-logo.svg" alt="All of Us"
-    style="height: 100%;">
+    style="height: 100%;" routerLink="/" routerLinkActive="active">
```

Comment:
How about wrapping the `<img>` in an `<a>` or a `<button>`? The advantage is that those elements come with some semantics for accessability / keyboard navigation.